lib/components/lenra_checkbox.dart

```diff
@@ -65,10 +72,21 @@ class LenraApplicationCheckbox extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    if(name != null) {
+      context.read<FormProvider>().setFormFieldValue(name!, value);
```

Comment:
What if there is a name without any form ? This will probably crash by returning null on `context.read<FormProvider>()`